uber/models.py

```diff
@@ -1048,7 +1049,10 @@ def _status_adjustments(self):
     @presave_adjustment
     def _staffing_adjustments(self):
         if self.ribbon == c.DEPT_HEAD_RIBBON:
-            self.staffing = self.trusted = True
+            self.staffing = True
+            # question: do we have info about what this person is a dept head of? or is that not something
+            # we know explicitly?  if so, we can either make them trusted in every dept, or trusted in no departments.
+            # self.trusted_depts = self.assigned_depts # this might be a reasonable thing to do. or not.
```

Comment:
business logic decision: 

this code is run when we save someone with a dept head ribbon.

before, assigning someone a dept head ribbon meant they were "trusted"

now: we have to decide which dept they are trusted with.  I see two options:
1) don't change anything i.e. potentially they're a dept head with no trusted depts. this is safe but maybe inconvenient
2) we could assign them to be a trusted staffer of any dept they are currently assigned to.  this could mean we generate some false positives i.e. we'd have people with dept head ribbons being marked as trusted in departments they actually aren't trusted in.  

There doesn't seem be any info about what this person is a dept head OF, just that they are a dept head, so there doesn't seem to be a way to choose a dept to make them trusted in.